tests/integration-tests/src/Scenario.ts

```diff
@@ -33,18 +34,21 @@ function writeOutput(api: Api, miniSecret: string) {
   // first and last key id used to generate keys in this scenario
   const keyIds = api.keyGenInfo()
 
+  const faucet = api.getFaucetInfo()
+
   const output: TestsOutput = {
     accounts,
     keyIds,
     miniSecret,
+    faucet,
   }
 
   fs.writeFileSync(OUTPUT_FILE_PATH, JSON.stringify(output, undefined, 2))
 }
 
 export async function scenario(label: string, scene: (props: ScenarioProps) => Promise<void>): Promise<void> {
   // Load env variables
-  config()
+  config({ path: './.env' })
```

Comment:
This is was left over from some testing I was doing, planning to add another line:
```
config({ path: '../../.env' })
```
To solve whole business of trying to get needed variables used for docker-compose services in a clean way into the scenario runner.